labs/accessing-optimizely-event-data/README.md

```diff
@@ -0,0 +1,140 @@
+
+```sh
+  $ oevents auth --token <optimizely personal access token>
+
+  export AWS_ACCESS_KEY_ID=<key id>
+  export AWS_SECRET_ACCESS_KEY=<secret access key>
+  export AWS_SESSION_TOKEN=<session token>
+  export AWS_SESSION_EXPIRATION=1594953226000
+  export S3_BASE_PATH=s3://optimizely-events-data/v1/account_id=12345/
+```
+
+## Exploring your Enriched Event data
+
+[decisions]: https://docs.developers.optimizely.com/web/docs/enriched-events-export#section-decisions
+[conversions]: https://docs.developers.optimizely.com/web/docs/enriched-events-export#section-conversions
+
+Enriched Events are partitioned into two top-level datasets, [decisions] and [conversions].  Each of these dataset is partitioned by date and experiment (for decisions) or event type (for conversions).  
+
+You can use `oevents ls` to list the all of the experiments that produced decision data on a given date:
+
+```sh
+$ oevents ls --type decisions --date 2020-05-10
+
+                           PRE experiment=10676850402/
+                           PRE experiment=14386840295/
+                           PRE experiment=14821050982/
+                           PRE experiment=15117030650/
+                           PRE experiment=17517981213/
+                           PRE experiment=17535310125/
+                           PRE experiment=8997901009/
+```
+
+You can also use `oevents` to list all of the event types collected on a given day:
+
+```sh
+$ oevents ls --type events --date 2020-05-10
+
+                           PRE event=search_query/
+                           PRE event=search_results_click/
+                           PRE event=add_to_cart/
+                           PRE event=purchase/
+```
+
+## Downloading your Enriched Event data
+
+You can use `oevents load` to download your Enriched Event data in [Apache Parquet](https://parquet.apache.org/) format.  Command line arguments can be used to specify a progressively narrower subset of your data.
+
+```sh
+$ oevents load --output ~/optimizely_data
+```
+
+will download *all* enriched event data associated with your Optimizely account.
+
+```sh
+$ oevents load \
+    --type decisions \
+    --output ~/optimizely_data
+```
+
+will download all [decision](decisions) data associated with your Optimizely account.
+
+```sh
+$ oevents load \
+    --type decisions \
+    --start 2020-07-01 \
+    --end 2020-07-05 \
+    --output ~/optimizely_data
+```
+
+will download all [decision](decisions) data collected between 7/1/2020 and 7/5/2020.
+
+```sh
+$ oevents load \
+    --type decisions \
+    --start 2020-07-01 \
+    --end 2020-07-05 \
+    --experiment 12345 \
+    --output ~/optimizely_data
+```
+
+will download all [decision](decisions) data collected between 7/1/2020 and 7/5/2020 for experiment #12345.
+
+## Testing `e3`
+
+You can run the `e3` test suite by install [BATS](https://github.com/bats-core/bats-core) and running `test.bats` in this directory.  Note: the `e3` test suite requires bash v4.4+.
```

Comment:
What is the `e3` test suite or what does it do?

Reply:
This was the old name for the CLI.  I replaced with `oevents`